src/components/CanvasSaveButton/README.md

```diff
@@ -0,0 +1,59 @@
+#
+
+This demonstrates the use of CanvasSaveButton.
+
+```jsx
+import React from 'react';
+import OLMap from 'ol/Map';
+import BasicMap from 'react-spatial/components/BasicMap';
+import ConfigReader from 'react-spatial/ConfigReader';
+import ShareMenu from 'react-spatial/components/ShareMenu';
+import LayerService from 'react-spatial/LayerService';
+
+class ShareMenuExample extends React.Component {
+  constructor(props) {
+    super(props);
+    this.map = new OLMap({ controls: [] });
+    this.center = [-10997148, 4569099];
+
+    const layers = ConfigReader.readConfig(
+      this.map,
+      treeData,
+    );
+
+    this.layerService = new LayerService(layers);
+  }
+
+  render() {
+    const extent = [-16681616.919511989, 655523.1517989757, -5312679.080488012, 8482674.848201025];
+
+    return (
+      <div className="tm-canvas-save-button-example">
+        <BasicMap
+          map={this.map}
+          center={this.center}
+          zoom={3}
+        />
+        <ShareMenu url={window.location.href}>
+          <CanvasSaveButton
+            title="Karte als Bild speichern."
+            className='tm-canvas-save-button tm-share-menu-icon'
+            map={this.map}
+            extent={extent}
+            extraData={{
+              copyright: '© Example copyright',
```

Comment:
it should be possible to get the copyright from the layerService like this:
```
copyright: layerService.getCopyrights()
```
Does it make sense?

Reply:
Can you try to set the copyright dynamically ? if a layers change is visibility the copyright must be updated.

Reply:
I had implemented this, see https://github.com/geops/react-spatial/pull/62/commits/12f9038758ad4a89258bf096b63ced95731ed8b6#diff-84392269f1dcee3ac19b57cd31741243L145. However, after I've talked to @mhaertwig, yesterday, I decided to simplify the properties. So the canvas save button is only responsible for writing on the canvas. Extracting the current copyright and retrieving the map rotation is not handled anymore by the canvas save button.

Reply:
@oterral I think we should keep the component simple. It's good enough to define the copyright manually here. 

However, if we want to reuse the functionality to display copyrights of active layers, we could add a `getActiveLayerCopyrights()` function to the `LayerService` that returns a list of visible layer copyrights.

Reply:
That's what I propose in my first comment. I just want to be sure it works , that the component is rerendered when a copyright change. 
Or even better, maybe just manage the case if this copyright property is a function.
```
extraData={{
    copyright: () => {
       return  layerService.getCopyrigths();
   }
}]

Reply:
> Changes I propose :
> 
>     * Add the possibility to specify a function or a number to the copyright property.
Implemented. Now you can pass a copyright function or a text.

I had re-reverted the static getCopyrights() function from the Copyright component. It is used in the CanvasSaveButtonExample. @oterral  suggested in the example above to define a getCopyrights() function in the LayerService. @edellucien @mhaertwig, should I leave it in the Copyright component or should I move it to the LayerService?

Reply:
Das Endergebnis ist die gleiche, aber ich denke auch, es wäre sauberer, es in layerService zu finden. Stimmst du auch dazu @mhaertwig ?